service/kernel/service_profiler.cpp

```diff
@@ -0,0 +1,46 @@
+/** file service_profiler.cpp */
+/*******************************************************************************
+* Copyright 2014-2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "services/internal/profiler.h"
+
+namespace daal
+{
+namespace services
+{
+namespace internal
+{
+
+daal::services::internal::ProfilerTask daal::services::internal::ProfilerDefault::startTask(const char *taskName)
```

Comment:
It makes sense to use short name, e.g., `ProfilerTask`, `ProfilerDefault`

Reply:
Done